sql_export/README.rst

```diff
@@ -0,0 +1,33 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+SQL Export
+==========
+
+Allow to export data in csv files FROM sql requests.
+There are some restrictions in the sql sql request, for example, it has to
+begin by "SELECT"
+A new menu named Export is created.
+
```

Comment:
The 'bug tracker' section is missing. (see https://github.com/OCA/maintainer-tools/tree/master/template/module#bug-tracker)